cinnabar/plotting.py

```diff
@@ -167,8 +172,14 @@ def _master_plot(
     # stats and title
     statistics_string = ""
     for statistic in statistics:
-        s = stats.bootstrap_statistic(x, y, xerr, yerr, statistic=statistic)
-        string = f"{statistic}:   {s['mle']:.2f} [95%: {s['low']:.2f}, {s['high']:.2f}] " + "\n"
+        s = stats.bootstrap_statistic(x,
+                                      y,
+                                      xerr,
+                                      yerr,
+                                      statistic=statistic,
+                                      bootstrap_experimental_uncertainty=bootstrap_experimental_uncertainty,
+                                      bootstrap_predicted_uncertainty=bootstrap_predicted_uncertainty)
+        string = f"{statistic}:   {s['mean']:.2f} [95%: {s['low']:.2f}, {s['high']:.2f}] " + "\n"
```

Comment:
Just an additional thought here, by using 'mean' for all cases here we do slightly move away from what's in the original issue (I think?).

Now we're saying that in all cases the estimate from the bootstrap samples rather than the original sample estimate is the thing we report. Assuming a statistic that is a linear function[note1] of the original data and enough bootstraps, I assume the two will come close but they are unlikely to be the same?

My understanding is that folks preferred using the sample estimate where possible (i.e. the case of adding noise is one of those cases where it's not possible). I'm not sure I have a formed opinion yet on which is better to be the most reported value[note2], that being said I do believe it to be important that we at least warn users of the difference between the two estimates, ensuring that folks are aware of any bias in the estimates.

[note1]: Are we guaranteeing this?
[note2]: I think I take the stance that ideally both would be reported when communicating science, but I do know it gets messy.

Reply:
>My understanding is that folks preferred using the sample estimate where possible (i.e. the case of adding noise is one of those cases where it's not possible).

I was unsure of this as well -- I thought @jchodera had mentioned in the call yesterday that in both the default and old modes, we want to use the mean instead of the MLE, but now that I'm thinking about it, he might have meant that when using either of the newly introduced bools are used, we want to use the mean. 

I will update the code so that in the default mode, the sample estimate (`s['MLE']`) is used. If one of the newly introduced bools is used, we used mean (`s['mean']`)

Reply:
Just pushed the changes

Reply:
If @jchodera has the time to review this it would be grand, it is one of those things where it seems like an innocuous change, but I do worry it will have wider consequences than I can see.

Maybe an opinion by @ppxasjsm would be good? That way we're a bit more sure we aren't messing up other's pipelines?